pkg/features/kube_features.go

```diff
@@ -663,6 +663,12 @@ const (
 	//
 	// Enables the usage of different protocols in the same Service with type=LoadBalancer
 	MixedProtocolLBService featuregate.Feature = "MixedProtocolLBService"
+
```

Comment:
Should also be in kubernetes/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go